tests/functional/conftest.py

```diff
@@ -21,11 +21,15 @@ def management(accounts):
     yield accounts[3]
 
 
-@pytest.fixture(params=["Normal", "NoReturn"])
+@pytest.fixture(
+    params=[("Normal", 18), ("NoReturn", 18), ("Normal", 8), ("NoReturn", 8)]
```

Comment:
I don't think we need to check `NoReturn` more than once, the behavior of that and decimals should be orthoganal. Also added a much lower value for decimals (like GUSD)
```suggestion
    params=[("Normal", 18), ("NoReturn", 18), ("Normal", 8), ("Normal", 2)]
```